src/app/core/utils/Input.ts

```diff
@@ -20,27 +20,48 @@ import {Event} from "./Events";
 
 export type Listener = (event: Event) => void;
 
+/**
+ * Class to handle user input, and trigger appropriate event listeners
+ */
 export class Input {
+    /** The canvas the user is performing inputs on */
     private canvas: HTMLCanvasElement;
+    /** A vector representing the previous position of the mouse*/
```

Comment:
You have a space before one of the `*/` but not the rest. You'll have to ask Leon which way it should be, but it at least should be consistent.